composer.json

```diff
@@ -105,13 +119,23 @@
         "allow-plugins": {
             "composer/installers": true,
             "drupal/core-composer-scaffold": true,
-            "drupal/core-project-message": true,
             "phpstan/extension-installer": true,
             "dealerdirect/phpcodesniffer-composer-installer": true,
             "cweagans/composer-patches": true
         },
         "sort-packages": true
     },
+    "scripts": {
+        "post-package-update": [
+            "Maestro\\Shell\\Events\\ComposerEventListener::postPackageUpdate"
+        ],
+        "post-install-cmd": [
+            "php -r \"copy('robots_custom.txt', 'web/robots.txt');\""
```

Comment:
This ensures the custom robots file is preserved after each composer install/update command. Took the corp-lite suffix away in favour of a general one (custom) for more re-usability. This single file might be a problem if each site needs its own but we can adjust to that later if that does become a requirement